resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -241,6 +250,57 @@ class MousePointer {
     }
 }
 
+[DSCResource()]
+class Scrollbars {
+
+    [DscProperty(Key)] [Status] $AlwaysShow = [Status]::KeepCurrentValue
+
+
+    hidden [string] $DynamicScrollbarsProperty = 'DynamicScrollbars'
+
+    [AlwaysShowScrollbars] Get() {
+        $currentState = [AlwaysShowScrollbars]::new()
+        
+		if (-not(DoesRegistryKeyPropertyExist -Path $global:ControlPanelAccessibilityRegistryPath -Name $this.DynamicScrollbarsProperty)) {
+            $dynamicScrollbarsValue = [BinarySettingState]::Disabled                
+        } else {
+			$dynamicScrollbarsValue = (Get-ItemProperty -Path $global:ControlPanelAccessibilityRegistryPath -Name $this.DynamicScrollbarsProperty).DynamicScrollbars
+			$currentState.ShowScrollBars = switch ($dynamicScrollbarsValue) {
+				1 { [BinarySettingState]::Disabled }               
+				0 { [BinarySettingState]::Enabled }               
+				default { [BinarySettingState]::KeepCurrentValue }
+			}
+				
+		}
+		
+        return $currentState
+    }
+
+    [bool] Test() {
+        $currentState = $this.Get()
+        if ($this.ShowScrollBars -ne [BinarySettingState]::KeepCurrentValue -and $this.ShowScrollBars -ne $currentState.ShowScrollBars) {
+            return $false
+        }
+
+        return $true
+    }
+
+    [void] Set() {
+        if ($this.ShowScrollBars -ne [BinarySettingState]::KeepCurrentValue) {
+            $desiredState = switch ([BinarySettingState]($this.ShowScrollBars)) {
+                Disabled { '1' }
+                Enabled { '0' }
+            }

```

Comment:
```suggestion
        if ($this.Test() -ne $true) {        
            $desiredState = $this.Show ? 0 : 1
        }

```

Reply:
Set-ItemProperty -Path $global:ControlPanelAccessibilityRegistryPath -Name $this.DynamicScrollbarsProperty -Value $this.DynamicScrollbarState